docs/getting-started.md

```diff
@@ -232,7 +224,682 @@ curl localhost:10101/index/repository/query \
+``` 
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/pilosa/go-pilosa"
+	"github.com/pilosa/go-pilosa/csv"
+	"io/ioutil"
+	"log"
+)
+
+func main() {
+	// Create the Schema
+	client := pilosa.DefaultClient()
+	schema, _ := client.Schema()
+	// This is where the index will go later
+	// This is where the fields will go later
+	err := client.SyncSchema(schema)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+```
+
+Next, let's create the `repository` index:
+```
+	repository := schema.Index("repository")
+```
+
+The index name must be 64 characters or less, start with a letter, and consist only of lowercase alphanumeric characters or `_-`. The same goes for field names.
+
+Let's create the `stargazer` field which has user IDs of stargazers as its rows:
+```
+	stargazer := repository.Field("stargazer")
+```
+
+Next up is the `language` field, which will contain IDs for programming languages:
+```
+	language := repository.Field("language")
+```
+
+Your `StarTrace.go` file should look like:
+```
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/pilosa/go-pilosa"
+	"github.com/pilosa/go-pilosa/csv"
+	"io/ioutil"
+	"log"
+)
+
+func main() {
+	// Create the Schema
+	client := pilosa.DefaultClient()
+	schema, _ := client.Schema()
+	repository := schema.Index("repository")
+	stargazer := repository.Field("stargazer")
+	language := repository.Field("language")
+	err := client.SyncSchema(schema)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+```
+
+##### Import Data From CSV Files
+
+Now that we have our index and our fields, we can import the data we downloaded earlier and be on our way to making our own queries.
+
+First, we will load our data into the `stargazer` field:
+```
+	stargazerFile, err := ioutil.ReadFile("stargazer.csv")
+	if err != nil {
+		log.Fatal(err)
+	}
+	format := "2006-01-02T15:04"
+	iterator = csv.NewColumnIteratorWithTimestampFormat(csv.RowIDColumnID, bytes.NewReader(stargazerFile), format)
+	err = client.ImportField(stargazer, iterator)
+	if err != nil {
+		log.Fatal(err)
+	}
+```
+Since our `stargazer` data contains time stamps, which represent the time users starred repos, we will be using the `csv.NewColumnIteratorWithTimeStampFormat` function that is built into the go-pilosa import. This function takes the format of the csv files (`csv.RowIDColumnID`), an `io.Reader` (`bytes.NewReader(stargazerFile)`), and the time quantum format (`format`) and translates the csv file into a format Pilosa can read. Time quantum is the resolution of the time we want to use.
```

Comment:
instead of "function that is built into the go-pilosa import", we can say "from the go-pilosa/csv package"2